### PR TITLE
add paradoc

### PR DESCRIPTION
**Pull request by Krande**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

### CODE REVIEW

**Comment by Krande** [comment]
@conda-forge/help-python, ready for review!

File: recipes/paradoc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Krande/paradoc/archive/v{{ version }}.tar.gz
+  sha256: b82ee4c9bc990e339f3341559dd9bd464efb3cf3c7105af5d8ef35920a0b03d4
+
+build:
+  skip: true  # [py<310]
```
Comment (ocefpaf) [inline recipes/paradoc/meta.yaml:13]: This recipe can probably be OS-noarch. See https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies

Comment (Krande) [reply recipes/paradoc/meta.yaml:13]: Hey @ocefpaf! Making it noarch was my original plan, but unfortunately I need the conditional windows run requirement `- pywin32 # [win]`. Do you know of a way I can make this noarch and still keep this conditional run requirement for windows users ?

(I don't want windows users having to separately install pywin32 for each paradoc installation )

**Comment by Krande** [comment]
@conda-forge-admin, please rerender

File: recipes/paradoc/meta.yaml (hunk 1 above)
Comment (Krande) [reply recipes/paradoc/meta.yaml]: Hey again, @ocefpaf. Looking at the conda-forge FAQ again, it seems I might have missed the `- __win   # [win]` requirement. I have tried building it as a noarch package, but it seems I will also have to modify the conda-forge yaml with 

```
noarch_platforms:
  - linux_64
  - win_64
```

which it seems I am not allowed to do according to the build bot?

File: recipes/paradoc/conda-forge.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+channel_priority: strict
+azure:
+  settings_win:
+    pool:
+      vmImage: windows-2022
```
Comment (ocefpaf) [inline recipes/paradoc/conda-forge.yml:5]:
```suggestion
```

Comment (Krande) [reply recipes/paradoc/conda-forge.yml]: done!

**Comment by Krande** [comment]
Hey @ocefpaf, it looks like the paradoc build failed after we merged the staged recipe so no package has been produced yet?

https://github.com/conda-forge/paradoc-feedstock/runs/28445374650

Got any ideas what might have gone wrong? Maybe I did something wrong in the conda-forge.yml file?

**Comment by ocefpaf** [comment]
See https://github.com/conda-forge/paradoc-feedstock/pull/2

**Pull request merged**